example/doc_view_acm_deconstruct.cpp

```diff
@@ -75,7 +75,7 @@ class TextView
   // give boost::signals2::deconstruct access to private constructor
   friend class boost::signals2::deconstruct_access;
   // private constructor to force use of deconstruct
-  TextView()
+  TextView() : m_document(0)
```

Comment:
There's a second instance of this that needs to be applied on line 115.

Reply:
It should be fixed now on develop branch.